### DISCUSSION TITLE
❗ Add a "headband" component to orbit.

### DISCUSSION THREAD

**Discussion by alexasselin008** [gsoft-inc/sg-orbit#1120] [New components Ideas]
## Description

In  many of  our products, we have a component that looks like this: 
![image](https://user-images.githubusercontent.com/38871812/194600938-fd6dfab7-f273-4c3d-a1e2-749be9535e2c.png)
![image](https://user-images.githubusercontent.com/38871812/194601081-645c91b3-f2f5-4a4a-9cb2-02435e2184bc.png)

**Comment by patricklafrance** [comment]
I suggest moving this to a Github discussion.
